### PR TITLE
Labour liverpool video styling

### PR DESCRIPTION
**Pull request by blongden73**
Adds some crazy angles to the videos when the labour and liverpool tag is on. 

HTML cleaner adds some new classes to the l&l videos, as well as adding some svgs within the video wrapper. 

Uses clip path to make the angles of the video, in IE this will just keep the video normal, as clip path is not supported. 

@SiAdcock @NataliaLKB could you take a look! 

picture 235
picture 236
picture 237

### CODE REVIEW

File: static/src/inline-svgs/membership/videoCaption.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+<svg viewBox="92 875 134 69" preserveAspectRatio="none">
+    <polygon id="" stroke="none" fill="#A4D8F4" fill-rule="evenodd" points="225.106952 880.342246 225.106952 920.395722 219.764706 925.737968 120.491979 925.737968 109.718449 943.545455 106.245989 943.545455 106.245989 925.737968 97.2532086 925.737968 92 920.395722 92 880.342246 97.342246 875 219.764706 875"></polygon>
```
Comment (SiAdcock) [inline static/src/inline-svgs/membership/videoCaption.svg]: Do we need `id`, `stroke`, `fill` and `file-rule` here? Can `fill` be specified in CSS?

File: static/src/stylesheets/inline/article-labour-liverpool.scss [hunk 2]
```diff
@@ -259,3 +259,70 @@ $labour-liverpool-header-element-opacity: .85;
         height: 88%;
     }
 }
+.content__article-body {
+    figure {
+        &.element-video--heroic {
```
Comment (SiAdcock) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: Is there any way to reduce this level of nesting? Would it be possible to just target the `.element-video--heroic` directly, without `.content__article-body` and `figure`?

File: static/src/stylesheets/inline/article-labour-liverpool.scss [hunk 3]
```diff
@@ -259,3 +259,70 @@ $labour-liverpool-header-element-opacity: .85;
...
+                //this is to align the bubble in the right position, countering the padding put on the caption-heroic
+                margin-left: -$gs-gutter/2;
+                margin-top: -$gs-gutter/2;
+            }
+
+            figcaption {
+                &.caption-heroic {
+                    position: absolute;
+                    top: 80%;
+                    left: -12px;
```
Comment (SiAdcock) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: Can this positioning be achieved using padding?

File: static/src/stylesheets/inline/article-labour-liverpool.scss (hunk 2 above)
Comment (blongden73) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: @SiAdcock if I un nest it does seem to go a bit mental. As there are classes elsewhere which are overriding. As they are more heavily nested.

Comment (NataliaLKB) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: Does that styling need to be deleted?

**Comment by SiAdcock** [comment]
👍 💎

File: static/src/inline-svgs/membership/videoCaption.svg (hunk 1 above)
Comment (NataliaLKB) [inline static/src/inline-svgs/membership/videoCaption.svg]: Was this addressed? Can those be removed in a separate PR?

File: static/src/inline-svgs/membership/videoCaption.svg [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+<svg viewBox="92 875 134 69" preserveAspectRatio="none">
```
Comment (NataliaLKB) [inline static/src/inline-svgs/membership/videoCaption.svg]: The viewbox doesn't look right here. It should almost always start with two `0`s. Also, it is normal to have a width and height. see: https://github.com/guardian/frontend/blob/master/docs/inline-svgs.md

File: static/src/stylesheets/inline/article-labour-liverpool.scss (hunk 2 above)
Comment (NataliaLKB) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: Was this addressed? Why so much nesting?

File: static/src/stylesheets/inline/article-labour-liverpool.scss [hunk 5]
```diff
@@ -259,3 +259,70 @@ $labour-liverpool-header-element-opacity: .85;
...
+
+            @include mq($until: phablet) {
+                margin-bottom: $gs-gutter*4;
+            }
+
+            @include mq($until: mobileLandscape) {
+                margin-bottom: $gs-gutter*5;
+            }
+
+            .gu-media-wrapper--video {
```
Comment (NataliaLKB) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: Again, does this need nesting?

File: static/src/stylesheets/inline/article-labour-liverpool.scss [hunk 6]
```diff
@@ -259,3 +259,70 @@ $labour-liverpool-header-element-opacity: .85;
...
+                -webkit-clip-path: polygon(0 6%, 100% 3%, 100% 100%, 0% 95%);
+                clip-path: polygon(0 6%, 100% 3%, 100% 100%, 0% 95%);
+                overflow: hidden;
+            }
+
+            .caption--img {
```
Comment (NataliaLKB) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: Should this be cleaned out instead using the html cleaner?

File: static/src/stylesheets/inline/article-labour-liverpool.scss [hunk 7]
```diff
@@ -259,3 +259,70 @@ $labour-liverpool-header-element-opacity: .85;
...
+                &::before {
+                    //this hides the little icon before the caption
+                    display: none !important;
+                }
+            }
+
+            .video-caption-bubble {
```
Comment (NataliaLKB) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: nesting?

File: static/src/stylesheets/inline/article-labour-liverpool.scss [hunk 8]
```diff
@@ -259,3 +259,70 @@ $labour-liverpool-header-element-opacity: .85;
...
+                width: 100%;
+                z-index: -10;
+                position: absolute;
+                //this is to align the bubble in the right position, countering the padding put on the caption-heroic
+                margin-left: -$gs-gutter/2;
+                margin-top: -$gs-gutter/2;
+            }
+
+            figcaption {
+                &.caption-heroic {
```
Comment (NataliaLKB) [inline static/src/stylesheets/inline/article-labour-liverpool.scss]: nesting?
